chromadb/utils/embedding_functions.py

```diff
@@ -1017,6 +1019,53 @@ def __call__(self, input: Documents) -> Embeddings:
         )
 
 
+
+
+class MlXEmbeddingFunction(EmbeddingFunction):
+
+    def __init__(
+            self,
+            bert_model,
+            weights_path,
```

Comment:
Let's improve on the DX here:

- Remove the param and make automatic conversion + caching of the model via a given cache path
- Add a conversion tool and make sure to describe how the conversion works so that devs can do the conversion themselves.